pocs/tests/utils/test_image_utils.py

```diff
@@ -21,6 +39,49 @@ def test_crop_data():
     assert cropped03.sum() == 36.
 
 
+def test_make_pretty_image(solved_fits_file, tiny_fits_file, save_environ):
+    # First make a dir and put the files in it
+    with tempfile.TemporaryDirectory() as tmpdir:
+        fz_file = os.path.join(tmpdir, os.path.basename(solved_fits_file))
+        fits_file = os.path.join(tmpdir, os.path.basename(tiny_fits_file))
+        # TODO Add a small CR2 file to our sample image files.
+
+        # Can't operate on non-existent files.
+        with pytest.warns(UserWarning):
+            assert not img_utils.make_pretty_image(fz_file)
+        with pytest.warns(UserWarning):
+            assert not img_utils.make_pretty_image(fits_file)
+
+        # Copy the files.
+        shutil.copy(solved_fits_file, tmpdir)
+        shutil.copy(tiny_fits_file, tmpdir)
+
+        # Not a valid file type (can't automatically handle fits.fz files).
+        with pytest.warns(UserWarning):
+            assert not img_utils.make_pretty_image(fz_file)
+
+        # Can handle the fits file, and creating the images dir for linking
+        # the latest image.
+        imgdir = os.path.join(tmpdir, 'images')
+        assert not os.path.isdir(imgdir)
+        os.environ['PANDIR'] = tmpdir
+
+        pretty = img_utils.make_pretty_image(fits_file, link_latest=True)
```

Comment:
I want to say that the issue arises when there is a stale symlink (perhaps with different permissions?) so I'm worried this won't fully cover the problem but it's definitely an improvement.